### PR TITLE
Fixed downloadable selected links price not added to product main price on cart configure page. #24579

### PR DESCRIPTION
**Pull request by rani-priya**
Fixed downloadable selected links price not added to product main price on cart configure page. #24579

### Fixed Issues (if relevant)

1. magento/magento2#24579: downloadable selected links price not added to product main price on cart configure page.

### Manual testing scenarios (*)

1. Create a downloadable product with some downloadable links
2. Select link and add to cart
3. edit that product from cart

### Questions or comments

downloadable

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Downloadable/view/frontend/web/js/downloadable.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 define([
     'jquery',
-    'jquery-ui-modules/widget',
+    'jquery/ui',
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/frontend/web/js/downloadable.js]: Why did you changed this line?
It was changed to `jquery-ui-modules/widget` in order to make split jquery UI to separate modules. I believe it's not related to the fix. If so - please revert this change

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @rani-webkul,
Please review my comment related to jquery ui change.

Also commit author's email is not related to your GitHub account (you can check that by clicking on icon near commit message, it should be clickable). Could you fix that by adding email to your github account OR changing commit author email & force push?

File: app/code/Magento/Downloadable/view/frontend/web/js/downloadable.js (hunk 1 above)
Comment (rani-priya) [reply app/code/Magento/Downloadable/view/frontend/web/js/downloadable.js]: Done, Actually it was by mistake.

**Comment by ihor-sviziev** [comment]
@rani-webkul author email still not related to your github account on 1st commit. 
https://github.com/magento/magento2/pull/24580/commits
image
Please fix that

**Comment by rani-priya** [comment]
@ihor-sviziev done

**Comment by ihor-sviziev** [comment]
Hi @rani-webkul, unfortunately commit still not related to your github account 😢

**Comment by rani-priya** [comment]
Hi @ihor-sviziev, Could you please let me know how I can fix this issue?
I'm facing this issue from initial days, and it's due to I had changed my username.

**Comment by ihor-sviziev** [comment]
@rani-webkul, please check your email in `git config --global user.email`. Looks like there is typo, there is no `@` symbol.
To change it execute following command `git config --global user.email your@email.com` where your@email.com - your email linked to GitHub account.

Just after change commit author and force push your changes

**Comment by rani-priya** [comment]
Hi @ihor-sviziev, Thank you.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-5837](https://jira.corp.magento.com/browse/ENGCOM-5837) has been created to process this Pull Request
:eight_spoked_asterisk: @ihor-sviziev, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**